README.md

```diff
@@ -5,24 +5,44 @@ This is an android application that is being created to gain experience developi
 applications written in Java, using git, solidifying programming fundamentals, and designing
 an application from markup to deployment.
 
+#### Code Rules
+  * SV Go shall be written in Java. 
+  * Application shall be written with strong variable typing.  
+     * This is to build good coding practices, and to have code written as implicit as possible. 
+  * Each line shall have a 100 character limit.  
+     * The is to improve code readability as it is
+   easier to read code in a digestible format.
+  * Code shall be properly formatted.  
+    * This is to maintain consistency throughout all of the code files.
+  * Every feature shall introduced will be thoroughly testing using unit testing as well as instrumented 
+  testing.  
+     * This is to improve the UX.
```

Comment:
Close - testing gives us confidence that our features are working as intended when we make bug fixes or new features.  Seeing x/x tests passed on build gives us confidence we've done our job right.